### PR TITLE
feat(approve): Add PR approval cmd

### PR DESCRIPTION
**Pull request by merkata**
# Summary

Adds the option to approve a PR, this is handy when working in a team and with having some branch protections in place (like require X approvals to merge).

## How did you test this change?

`go run ./gh-dash.go --debug`

## Images/Videos

gh_dash_approve

### CODE REVIEW

File: ui/components/prsidebar/approve.go
Comment (dlvhdr) [inline ui/components/prsidebar/approve.go:20]: Let's not include the content of the comment as it can be quite long and have new lines. This will break the UI once rendered

File: ui/keys/prKeys.go
Comment (dlvhdr) [inline ui/keys/prKeys.go:29]: I think <kbd>v</kbd> would work better here? as in a checkmark?
maybe we can change to <kbd>a</kbd> when we introduce a breaking change.

**Review by dlvhdr — COMMENTED** [review]
This is great! Thanks for contributing :)
Had some minor comments

File: ui/keys/prKeys.go
Comment (merkata) [reply ui/keys/prKeys.go:29]: That's great, changed!

File: ui/components/prsidebar/approve.go
Comment (merkata) [reply ui/components/prsidebar/approve.go:20]: Makes sense, adjusted. Thanks!

**Review by dlvhdr — APPROVED** [review]

File: ui/components/prsidebar/prsidebar.go
Comment (rmacklin) [inline ui/components/prsidebar/prsidebar.go:87]: This seems to require a comment to be entered, or else the approval won't occur. Yet the code here:
https://github.com/merkata/gh-dash/blob/d9033f7855730e536f5a3ecfbd8cb8e9f4c9268b/ui/components/prsidebar/approve.go#L33-L35
seems to indicate that approving with no comment is supposed to be supported. Do we want to support approving without a comment (like this PR itself was approved: https://github.com/dlvhdr/gh-dash/pull/399#pullrequestreview-2149546111)?

Comment (rmacklin) [reply ui/components/prsidebar/prsidebar.go:87]: I've opened a PR to enable approving without a comment:
- https://github.com/dlvhdr/gh-dash/pull/420

**Pull request merged**
